### PR TITLE
feat(data dictionary): Add K8sCronjobSample

### PR DESCRIPTION
**Pull request by htroisi**
We recently added the `K8sCronjobSample` event to New Relic. This PR adds that info to the data dictionary.

### CODE REVIEW

File: src/data-dictionary/events/K8sCronjobSample/label.LABEL_NAME.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: label.LABEL_NAME
+type: attribute
+units:
+events:
+  - K8sCronjobSample
+---
+
+Labels associated with your CronJob, so you can filter and query for specific CronJobs.
```
Comment (ally-sassman) [inline src/data-dictionary/events/K8sCronjobSample/label.LABEL_NAME.md:9]:
```suggestion
Labels associated with your CronJob so you can filter and query for specific CronJobs.
```

File: src/data-dictionary/events/K8sCronjobSample/lastScheduledTime.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: lastScheduledTime
+type: attribute
+units:
+events:
+  - K8sCronjobSample
+---
+
+The last time the CronJob successfully scheduled a Job.
```
Comment (ally-sassman) [inline src/data-dictionary/events/K8sCronjobSample/lastScheduledTime.md:9]:
```suggestion
The last time the CronJob successfully scheduled a job.
```

File: src/data-dictionary/events/K8sCronjobSample/nextScheduledTime.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+name: nextScheduledTime
+type: attribute
+units:
+events:
+  - K8sCronjobSample
+---
+
+Next time the Cronjob should be scheduled. The time after lastScheduleTime, or after the Cronjob's creation time if it's never been scheduled. Use this to determine if the job is delayed.
```
Comment (ally-sassman) [inline src/data-dictionary/events/K8sCronjobSample/nextScheduledTime.md:9]:
```suggestion
Next time the Cronjob should be scheduled. The time after `lastScheduleTime`, or after the Cronjob's creation time if it's never been scheduled. Use this to determine if the job is delayed.
```

File: src/data-dictionary/events/K8sCronjobSample/schedule.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+name: schedule
+type: attribute
+units:
+events:
+  - K8sCronjobSample
+---
+
+The schedule which the CronJob uses to run Jobs.
```
Comment (ally-sassman) [inline src/data-dictionary/events/K8sCronjobSample/schedule.md:9]:
```suggestion
The schedule which the CronJob uses to run jobs.
```

File: src/data-dictionary/events/K8sCronjobSample/K8sCronjobSample.event-definition.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+name: K8sCronjobSample
+type: event
+dataSources:
+  - Kubernetes
+---
+
+This event contains Kubernetes `CronJob` data.
```
Comment (ally-sassman) [inline src/data-dictionary/events/K8sCronjobSample/K8sCronjobSample.event-definition.md:8]: @htroisi I think we should format all CronJob mentions be in code format (`CronJob`), or all are regular (CronJob) to be consistent, so consider editing this to just CronJob to match the other references.

Comment (htroisi) [reply src/data-dictionary/events/K8sCronjobSample/K8sCronjobSample.event-definition.md]: Good point. I'll fix it

**Pull request merged**